insights/parsers/ls_var_opt_mssql.py

```diff
@@ -0,0 +1,30 @@
+"""
+LsVarOptMSSql - command ``ls -laR /var/opt/mssql``
+==================================================
+
+The ``ls -laR /var/opt/mssql`` command provides information for the listing of
+the ``/var/opt/mssql`` directory. See ``FileListing`` class for addtional information.
+
+
+Sample ``ls -laR /var/opt/`` output::
```

Comment:
Should be `/var/opt/mssql`?

Reply:
@lonicerae Looks like I missed the full path. Fixed in the latest commit.